docs/clients/brushless_drive.rst

```diff
@@ -105,72 +105,116 @@ Message Table
+| 40     | motor_l                       | get, set, save | float     | H        | Cross inductance                                                                                                                     |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 41     | derate                        | get            | int32     | PU fix16 | Amount of derating. No derate = 65536, full derate = 0                                                                               |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 44     | emf                           | get            | float     |          |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 45     | volts_at_max_amps             | get            | float     |          |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 46     | slew_volts_per_second         | get, set, save | float     | V        |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 47     | slew_enable                   | get, set, save | float     | V        |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 48     | motoring_supply_current_limit | get, set, save | float     | fix16_t  |                                                                                                                                      |      |
```

Comment:
Description: "On modules that support dynamic supply current limiting, this entry determines the current limit for the current from the supply when the module is "motoring", i.e. the drive voltage is greater than the back-EMF, as opposed to when it is regenerating. Note this is the current from the supply, so it would be the current from your battery or power supply."

Units are A